nix/tools/tests.nix

```
@@ -24,7 +24,31 @@ let
         withEnv = postgrest.env;
       }
       ''
-        ${withTools.latest} ${cabal-install}/bin/cabal v2-test ${devCabalOptions}
+        ${withTools.latest} ${cabal-install}/bin/cabal v2-run ${devCabalOptions} test:spec
+      '';
+
+  testSpecQuerycost =
+    checkedShellScript
+      {
+        name = "postgrest-test-spec-querycost";
```

```suggestion
  testQuerycost =
    checkedShellScript
      {
        name = "postgrest-test-querycost";
```

Allows for better autocompletion, because you only have to do continue with `s`, `q` or `d` at `postgrest-test-` once.

Maybe change that everywhere for consistency.

---

yes, makes sense! fixed